src/Nowruz/modules/credentials/KYB/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { DescriptionModal } from './descriptionModal';
+import { UploadModal } from './uploadModal';
+import { SuccessModal } from './successModal';
+
+interface KYBModalProps {
+  open: boolean;
+  setOpen: (val: boolean) => void;
+}
+export const KYBModal: React.FC<KYBModalProps> = ({ open, setOpen }) => {
+  const [openDescModal, setOpenDescModal] = useState(true);
+  const [openUploadModal, setOpenUploadModal] = useState(false);
+  const [openSuccessModal, setopenSuccessModal] = useState(false);
+  const handleClose = () => {
+    setOpenDescModal(false);
```

Comment:
and here it will be easy for you to write one line code `setOpenModal({...openModal, open:false})`